pkg/apis/configuration/validation/validation_test.go

```diff
@@ -1378,3 +1381,59 @@ func TestValidateVirtualServerRouteSubroutesFails(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateUpstreamLBMethod(t *testing.T) {
+	tests := []struct {
+		method string
+		isPlus bool
+	}{
+		{
+			method: "round_robin",
+			isPlus: false,
+		},
+		{
+			method: "",
+			isPlus: false,
+		},
+		{
+			method: "ip_hash",
+			isPlus: true,
+		},
+		{
+			method: "",
+			isPlus: true,
+		},
+	}
+
+	for _, test := range tests {
+		allErrs := validateUpstreamLBMethod(test.method, field.NewPath("lb-method"), test.isPlus)
+
+		if len(allErrs) != 0 {
+			t.Errorf("validateUpstreamLBMethod() returned errors for method %s", test.method)
```

Comment:
Could you possibly also print out the value of isPlus variable. Because the signature of the function is simple, I suggest something like this:
validateUpstreamLBMethod(%q, %v) returned ...
Note that I used %q to make sure when the method is empty, the empty string is printed like “”. 
This comments also applies for line 1436.